src/Symfony/Component/ErrorHandler/Exception/SilencedErrorContext.php

```diff
@@ -34,27 +34,27 @@ public function __construct(int $severity, string $file, int $line, array $trace
         $this->count = $count;
     }
 
-    public function getSeverity()
+    public function getSeverity(): int
     {
         return $this->severity;
     }
 
-    public function getFile()
+    public function getFile(): string
     {
         return $this->file;
     }
 
-    public function getLine()
+    public function getLine(): int
     {
         return $this->line;
     }
 
-    public function getTrace()
+    public function getTrace(): array
     {
         return $this->trace;
     }
 
-    public function JsonSerialize()
+    public function JsonSerialize(): array
```

Comment:
wrong case. should be `jsonSerialize`

Reply:
we must fix that for Debug component too in 3.4

Reply:
ok -> https://github.com/symfony/symfony/pull/32826